### ISSUE TITLE
[Feedback] Verscheidene Sachen - Webseitenliste im menu ...  

### ISSUE THREAD

**Issue by gsegit** [iakmds/buhocms#5]
Hallo ;D
Also mir sind ein paar Sachen aufgefallen, weiss nicht ob das ein Fehler ist, daher schreibe ich mal hier rein und dazu noch ein Vorschlag.Wenn man die Webseite öffnen will, dann kommt bei jedem klicken eine neue Seite, wäre super der Server nur ein Reload machen würde. Bei manchen Themes hat deine App immer noch Probleme, kann man nicht was einbauen, womit man das Theme testen kann, ob es überhaupt brauchbar ist ?
Und als Vorschlag, vielleicht eine Liste von den Webseiten, die man schon geöffnet wurden, so das man nicht immer in die Schritte nochmal machen muss.

Das wärs eigentlich ;D

Tschöö

Bogus

**Comment by iakmds** [comment]
> Wenn man die Webseite öffnen will, dann kommt bei jedem klicken eine neue Seite, wäre super der Server nur ein Reload machen würde

Wie genau meinst du das?

> Bei manchen Themes hat deine App immer noch Probleme, kann man nicht was einbauen, womit man das Theme testen kann, ob es überhaupt brauchbar ist ?

Ich weiß nicht, ob das mit BuhoCMS zu tun hat, oder ob man auch so einige Einstellungen vornehmen muss (wenn man manuell Themes auswählt)? In BuhoCMS ist auf der Seite nach dem runterladen der Themes eine Nachricht zu sehen, die sagt `Denke daran, die Dokumentation im Themen-Repository zu besuchen, um Details zur Implementierung der richtigen Konfigurationen zu erfahren.` Diese Narchicht habe ich jetzt nach links beim auswählen des Themes gepackt, anstatt dass sie nur nach dem runterladen zu sehen ist.

> Und als Vorschlag, vielleicht eine Liste von den Webseiten, die man schon geöffnet wurden, so das man nicht immer in die Schritte nochmal machen muss.

Das ist ein guter Vorschlag! :+1:
Meinst du das z.B. so, dass wenn man auf "Webseite öffnen" drückt, man direkt eine Liste der vorherigen Ordner sieht, anstatt immer wieder auf 1. "Weiter", 2, "Ordner auswählen", 3. "Öffnen" drücken zu müssen?

**Comment by gsegit** [comment]
Mit dem Reload meinte ich, wenn ich mir die Webseite im Browser anschaun möchte, kommt jedes mal ein neues Fenster, anstatt ein Reload, was der Hugo server normal machen würde. Vielleicht könnte man unter dem Webseite im Browser öfffnen noch ein Icon Reload machen ?
Das mit dem Themes ist irgendwie sehr schwierig, weil man nicht weiss was für ein Config Format benutzt wird. Das macht für mich das ganze schwieriger. Das mit dem Konfig exportieren und importieren ist aber sehr nützlich

**Comment by iakmds** [comment]
> Mit dem Reload meinte ich, wenn ich mir die Webseite im Browser anschaun möchte, kommt jedes mal ein neues Fenster, anstatt ein Reload, was der Hugo server normal machen würde. Vielleicht könnte man unter dem Webseite im Browser öfffnen noch ein Icon Reload machen ?

Achso, jetzt verstehe ich es. Ich weiß nicht, ob sich noch ein Button lohnt, da man ja auch einfach das Browserfenster aktualisieren kann (F5)

> Das mit dem Themes ist irgendwie sehr schwierig, weil man nicht weiss was für ein Config Format benutzt wird. Das macht für mich das ganze schwieriger. Das mit dem Konfig exportieren und importieren ist aber sehr nützlich

Wie sind deine Erfahrungen bzgl. Themes ohne BuhoCMS? Also bezieht sich das mit den Themes auf BuhoCMS oder mehr auf Hugo allgemein?

> Würde gerne diesen Thread für allgemeinen Feedback nutzen, ist das ok ?

Klar

**Comment by gsegit** [comment]
> Wie sind deine Erfahrungen bzgl. Themes ohne BuhoCMS? Also bezieht sich das mit den Themes auf BuhoCMS oder mehr auf Hugo allgemein?

Hmm ... Ich hab noch nicht soviel Erfahrung mit Themes, glaube aber eher das Problem die Submodules sind. Ich versuch mal ein eigenes Theme zu erstellen bzw. ein erstmal einfaches.

**Comment by iakmds** [comment]
Übrigens, eine neue Version ist jetzt da (0.3.0) :tada: 
Einige der neuen Features:

- Beim Öffnen einer Webseite siehst du jetzt die zuletzt geöffneten Webseiten
- Ein neuer Button ("Kommandozeilen-Ausgabe", in der linken Navigationsleiste) zeigt jetzt die Ausgabe der einzelnen Befehle, mit der wir vielleicht die Fehler genauer analysieren können ;)
- BuhoCMS ist jetzt auch als .deb Paket :package: erhältlich

**Comment by gsegit** [comment]
So, ich hab irgendwie ein Verständnisproblem oder bin zu blöd. Jedenfalls kriege ich nicht hin, das ich meine Artikel auf der erstellten Webseite angezeigt werden, warum auch immer .. 
Hier mal die Schritte, die ich mache um eine neue Webseite zu erstellen
1, Webseite erstellen im Menu auswählen
2- Einstellungen, Hugo Themes auswählen bzw. hochladen
3. Artikel Schreiben .. Draft auf False ?!
4. Hugo Server starten
5. Webseiten öffnen

Das wärs, dann sehe ich die Webseite  aber halt ohne Artikel Ich werde mal per Console eine webseite erstellen und dann einladen, bin gespannt ob das geht ;D

**Comment by gsegit** [comment]
So nun hab ich mal nach 
https://gohugo.io/getting-started/quick-start/
eine Webseite erstellt und ich sehe wieder mein Artikel nicht auf der Webseite, wobei der Artikel in einem Ordner posts liegt. Das dürfte ja eigentlich kein Problem sein oder ? 
Da wird aber nach dem einladen des Themes, ein 
`echo "theme = 'ananke'" >> config.toml` 
Oder liegt es daran, das hier *.toml benutzt wird

Das Theme, was da benutzt wird,  sieht übrigens super aus ;D

**Comment by gsegit** [comment]
mähhh, hab den fehler gefunden, draft muss false sein ... dann wird auch der Artikel angezeigt, wobei ich nicht ganz verstehe, was dann das Draft bedeutet .. lol

**Comment by iakmds** [comment]
> mähhh, hab den fehler gefunden, draft muss false sein ... dann wird auch der Artikel angezeigt, wobei ich nicht ganz verstehe, was dann das Draft bedeutet .. lol

Genau, `draft` bedeutet so viel wie "Entwurf", weshalb der auf `false` sein muss, wenn man den Artikel sehen will :smile:

**Comment by gsegit** [comment]
**Vorschlag:**
Vielleicht wäre in der Readme.md ein Link zu AppImagesLaucher nicht schlecht ;D
https://github.com/TheAssassin/AppImageLauncher

**Comment by iakmds** [comment]
Ist schon vorhanden :smile: 

![image](https://user-images.githubusercontent.com/111225446/225443530-c55a600b-ff13-4002-aef0-0302d952e669.png)

**Comment by gsegit** [comment]
Als ich die deb Package installiert hab, hab ich gesehen das deine App als nicht offen gekennzeichnet wurde und da fehlen auch Screenshots. Hab die App über einen Launcher installiert ;D Und eine gannnnzzzz grosse bitte, schreib bei den Packages die Versionsnummer in den Dateinamen  ;D 
Was die ThemesThema angeht, hab bisher kein Theme gefunden was nicht irgendwie Modules oder sonst irgendwie was besonderes drin hat. Es gibt aber echt sehr schöne Themes ;D Werde mal am WE einige testen und schaun wie die mit der APP laufen und hier berichten ;D

**Comment by iakmds** [comment]
> Als ich die deb Package installiert hab, hab ich gesehen das deine App als nicht offen gekennzeichnet wurde und da fehlen auch Screenshots. Hab die App über einen Launcher installiert ;D

Inwiefern? Also wo wurde sie als nicht offen gekennzeichnet? Und wo genau fehlen die Screnshots, im Launcher?

>  Und eine gannnnzzzz grosse bitte, schreib bei den Packages die Versionsnummer in den Dateinamen ;D

Ja, macht eigentlich Sinn... ändere mal die vorherigen, die nächsten werden so kommen. Danke!

**Comment by gsegit** [comment]
Bist du LinuxUser ? Jedenfalls wird in dem App Launcher, ich nutze da Ubuntu Budgie 22.10. in dem launcher wird deine APP als Proprietär bezeichnet, also nicht OpenSource. Und die Screenshots, über der Beschreibung werden meist so Screenshots von den APPs gezeigt, da waren keine. Ich versuche mal mal ein Screenshot zu machen ;D

**Comment by iakmds** [comment]
Ja, ich benutze Linux Mint. Meinst du mit App Launcher praktisch den Software Store, also wo man Programme installiert? Falls ja, liegt das wahrscheinlich daran, dass die app zwar im Store erkannt wird, das aber nur, weil die app bereits installiert ist - sie wird zwar erkannt, ist aber nicht wie die normalen Pakete in den Paketquellen vorhanden, weil es eine externe .deb-Datei ist. Deswegen werden wahrscheinlich keine Informationen vorgestellt

**Comment by gsegit** [comment]
Ist irgendwie witzig, ich nutze auch Linux Mint ;D 
Ja, das mit den Infos und launcher, da kannst recht haben.

**Comment by gsegit** [comment]
Also irgendwie ist bei mir der Wurm, ich hab wieder das Problem das ich die Postings, also die Artikel von mir nicht auf der Seite sehen kann. Keine Ahnung woran es liegt, weil es wird keine Fehlermeldung ausgegeben und Draft steht auch auf False. Ich schau mal am WE, ob ich den fehler finde. Denke es wird wieder ein Anwendungsfehler von mir sein. 
Was für ein Theme nutzt du denn ?

**Comment by iakmds** [comment]
Als Test benutze ich oft das hier: https://github.com/adityatelange/hugo-PaperMod

**Comment by gsegit** [comment]
Also ich hab auch das was da steht als Option genommen, aber trotzdem wird nichts angezeigt, ist wie verhext ;( 

Was mir aber noch aufgefallen ist, in dem Textfeld sehe ich keinen Cursor. Ich klicke mit der Maus rein, aber irgendwie sehe ich nicht wo der Cursor grade ist.

**Comment by iakmds** [comment]
Das ist extra so eingestellt, also dass nur die Inhalte des Content folders zu sehen sind, da man eigentlich nur die Artikel bearbeiten können soll, nicht jedoch alles rund um Hugo (dafür kann man wie sonst einen Text-Editor verwenden, man kann z.B. einen Rechtsklick auf das leere Feld bei den Dateien in BuhoCMS machen und dann auf "Im Dateiordner öffnen")

**Comment by gsegit** [comment]
Ja hast irgendwie recht, geht ja um den content 

Die einfache seite, die ich nun erstellt hab, hat übriegens kein theme. Da wird nur der eine content datei angezeigt. Würde echt wissen, wo ich den fehler mach .. ok als nächstes binde ich ein theme ..  schaun wir mal 😃

**Comment by gsegit** [comment]
Halo ;D
Hab in moment einige Probleme mit meinen blöden DSL Provider, daher kann ich leider nicht so agieren wie ich möchte. Bin deshlab ein wenig still ;D

**Comment by gsegit** [comment]
Moin ;D
Hast du mitbekommen, das ab der aktuellen Version von Hugo TailwindCSS unterstützt wird?

**Comment by iakmds** [comment]
Hi,
ja, ich glaube das habe ich gelesen

**Comment by gsegit** [comment]
Hallo ;D
Mir sind noch ein paar Sachen eingefallen, die du vielleicht einbauen könntest ;D
- als Option git intrigieren
- Theme editieren / CSS dateien intrigieren

**Comment by gsegit** [comment]
Hey ;D Machst du zur Zeit ne Pause ?

**Comment by iakmds** [comment]
> Hey ;D Machst du zur Zeit ne Pause ?

Hey, ja könnte man sagen... Die Motivation für dieses Projekt hat etwas nachgelassen, momentan arbeite ich an einem anderen Projekt, das ich bald veröffentlichen möchte. Ich weiß nicht genau, wann ich wieder zu BuhoCMS zurückkomme, jedenfalls danke ich dir bis dahin für all deine Ideen!
